client/lib/analytics/README.md

```diff
@@ -7,118 +7,21 @@ Turn on debugging in the JavaScript developer console to view calls being made w
 
 `localStorage.setItem('debug', 'calypso:analytics:*');`
 
-You can limit to only calls made to GA, Tracks, or MC by replacing the `*` with an appropriate suffix. `ga` for Google Analytics, `tracks` for Tracks, and `mc` for MC.
+You can limit to only calls made to Google Analytics, Tracks, or MC by replacing the `*` with an appropriate suffix. `ga` for Google Analytics, `tracks` for Tracks, and `mc` for MC.
 
 `localStorage.setItem('debug', 'calypso:analytics:tracks'); // only show debug for tracks`
 
 
 ## Which analytics tool should I use?
 
-*Page Views* (`analytics.pageView.record`) should be used to record all page views (when the main content body completely changes). This will automatically record the pageview to both Google Analytics and Tracks.
+_Note: Each tool has its own documentation with extensive usage guidelines and examples._
 
-*Google Analytics* should be used to record all events the user performs on a page that *do not* trigger a page view (this will allow us to determine bounce rate on pages).
+[**Page Views**](./docs/page-views.md) should be used to record all page views (when the main content body completely changes). This will automatically record the page view to both Google Analytics and Tracks.
 
-We are using GA to monitor user flows through the user interface of Calypso in order learn where they succeed and fail, as well as determine usage of different sections. *Please do not ship anything that does not have GA tracking in place*, otherwise we will create big gaps in our understanding.
+[**Google Analytics**](./docs/google-analytics.md) should be used to record all events the user performs on a page that *do not* trigger a page view (this will allow us to determine bounce rate on pages).
```

Comment:
This one is from before but I'm actually confused by what it means. We use GA to track page views, but we should also track events on pages that don't record a page view? When should we use GA over tracks (which we use for funnels)? Or does this mean we should always record to both?

@yanirs would it be possible for you clarify here?

Reply:
I'm not sure. @markryall and @Essayoh have been working on GA tracking recently so they might be able to help.

Reply:
I think all of this is being revisited and we will drive future GA changes out of gaps we can see in the current dashboard or other currently unused GA features.  The intention is to find the best use cases for tracks vs GA.

Reply:
Since I'm not familiar with this, I'll leave GA descriptions as-is, and ready to be improved by someone who actually knows about them. 🙂